main.py

```diff
@@ -1,93 +1,40 @@
-from module.create import create_folder, create_logger
```

Comment:
I like the idea of allowing the user to run either cli or the gui....Great work 👍